apps/gp2-frontend/src/users/__tests__/api.test.tsx

```diff
@@ -95,6 +95,48 @@ describe('getUsers', () => {
   });
 });
 
+describe('patchUser', () => {
+  it('makes an authorized PATCH request for the user id', async () => {
+    const patch = { firstName: 'New Name' };
+    nock(API_BASE_URL, { reqheaders: { authorization: 'Bearer x' } })
+      .patch('/users/42')
+      .reply(200, {});
```

Comment:
How does this work with the nock?

In the other describe blocks we have a nock.cleanAll? Shouldn't we need this here?

Reply:
I just did the same test as crn, which I mostly for coverage, we are just testing that a call to the right endpoint with the right arguments is made